### ISSUE TITLE
Продължаващи проблеми с ГИТ процеса

### ISSUE THREAD

**Issue by yavorp** [NikoletaDalaktchieva/instant-messenger#7]
Не се оставят ревюта при Pull Request. Всеки отворен Pull Request е добре да бъде ревюуиран и експлицитно одобрен, за да може повече от 1 човек да се запознае с кода, за да се допускат по-малко грешки. Ако Pull Request не е експлицитно одобрен не е добре да бъде сливан както и ако има коментари, които не са резолвнати

**Comment by NikoletaDalaktchieva** [comment]
Да, вчера просто имаше проблем с angularp-а и трябваше да обединим на всички частите, за да може всеки да види как да оправи своята и да се нагодим. Когато всичко е готово ще направим pr към main, за да може да го прегледате :) Обърнали сме внимание на коментарите и ще ги имаме в предвид преди да направим pr към main-а. Извиняваме се и за в бъдеще ще гледаме да не се повтаря.
